t.go

```diff
@@ -203,3 +218,157 @@ func (t *Torrent) AddTrackers(announceList [][]string) {
 	defer t.cl.mu.Unlock()
 	t.addTrackers(announceList)
 }
+
+func (t *Torrent) RemoveTracker(addr string) {
+	t.cl.mu.Lock()
+	defer t.cl.mu.Unlock()
+
+	nn := make([][]string, 0)
+
+	for _, tier := range t.metainfo.AnnounceList {
+		n := make([]string, 0)
+		for _, tracker := range tier {
+			if tracker != addr {
+				n = append(n, tracker)
+			} else {
+				if trackerScraper, ok := t.trackerAnnouncers[tracker]; ok {
+					trackerScraper.stop.Set()
+				}
+			}
+		}
+		if len(n) > 0 {
+			nn = append(nn, n)
+		}
+	}
+
+	t.metainfo.AnnounceList = nn
+}
+
+type Tracker struct {
+	Url          string
+	Peers        int
+	Err          string
+	LastAnnounce int64
+	NextAnnounce int64
+}
+
+func (t *Torrent) Trackers() []Tracker {
+	t.cl.mu.Lock()
+	defer t.cl.mu.Unlock()
+
+	var tt []Tracker
+
+	for _, tier := range t.announceList() {
+		for _, trackerURL := range tier {
+			T := Tracker{Url: trackerURL}
+			if trackerScraper, ok := t.trackerAnnouncers[trackerURL]; ok {
+				T.Peers = trackerScraper.peers
+				T.Err = trackerScraper.err
+				T.LastAnnounce = trackerScraper.lastAnnounce
+				T.NextAnnounce = trackerScraper.nextAnnounce
+			}
+			tt = append(tt, T)
+		}
+	}
+
+	tt = append(tt, Tracker{"DHT", t.peersDHT, t.errDHT, t.lastAnnounceDHT, 0})
```

Comment:
These should be gotten from Torrent.Stats, rather than forcing them to fit into Tracker stats